### PR TITLE
Update getting_started.zh-cn.md

### PR DESCRIPTION
**Pull request by iamxiaojianzheng**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/grid/getting_started.zh-cn.md [hunk 1]
```diff
@@ -214,113 +203,83 @@ driver.quit();
...
 
-API calls can be done through the [http://localhost:4444/status](http://localhost:4444/status)
-endpoint or using [GraphQL]({{< ref "advanced_features/graphql_support.md" >}})
+API 调用可以通过 [http://localhost:4444/status](http://localhost:4444/status) 端点或使用 [GraphQL]({{< ref "advanced_features/graphql_support.md" >}})
 
 {{% pageinfo color="primary" %}}
-For simplicity, all command examples shown in this page assume that components are running
-locally. More detailed examples and usages can be found in the
-[Configuring Components]({{< ref "/configuration" >}}) section.
+为简单起见，本页中显示的所有命令示例均假定组件正在运行
```
Comment (alaahong) [inline website_and_docs/content/documentation/grid/getting_started.zh-cn.md]: so, are you translating line by line? Seems it should be treating as "running locally" ~~~

File: website_and_docs/content/documentation/grid/getting_started.zh-cn.md [hunk 2]
```diff
@@ -214,113 +203,83 @@ driver.quit();
...
-requests and responses asynchronously. Additionally it also provides WebSocket support. Hence it 
-is a good fit. 
+默认情况下，Grid 将使用 [AsyncHttpClient](https://github.com/AsyncHttpClient/async-http-client)。 AsyncHttpClient 是一个建立在 Netty 之上的开源库。 它允许异步执行 HTTP 请求和响应。 此外，它还提供 WebSocket 支持。 因此它很合适。
 
-However, AsyncHttpClient is not been actively maintained since June 2021. It coincides with the 
-fact that Java 11+ provides a built-in HTTP and WebSocket client. Currently, Selenium 
-has plans to upgrade the minimum version supported to Java 11. However, it is a sizeable effort. 
-Aligning it with major releases and accompanied announcements  is crucial to ensure the user 
-experience is intact.
+然而，AsyncHttpClient 从 2021 年 6 月开始就没有主动维护了。恰逢 Java 11+ 提供了内置的 HTTP 和 WebSocket 客户端。 目前，Selenium 计划将支持的最低版本升级到 Java 11。但是，这是一个相当大的努力。 将其与主要版本和随附的公告保持一致对于确保用户体验完好无损至关重要。
```
Comment (alaahong) [inline website_and_docs/content/documentation/grid/getting_started.zh-cn.md]: sizeable effort? I think that means we have to do many works, not means try much?

File: website_and_docs/content/documentation/grid/getting_started.zh-cn.md [hunk 3]
```diff
@@ -214,113 +203,83 @@ driver.quit();
...
 
-**Hub/Node** between 60 and 100 **Nodes**. **Distributed** with over 100 **Nodes**.
+**Hub/Node** 介于60到100个 **Nodes** 之间， **Distributed** 超过100个 **Nodes**。
 
 ## Warning
 
-Selenium Grid must be protected from external access using appropriate
-firewall permissions.
+必须使用适当的方法保护 Selenium Grid 免受外部访问
+防火墙权限。
```
Comment (alaahong) [inline website_and_docs/content/documentation/grid/getting_started.zh-cn.md]: Can you translate within one scenario?  You treat them as "visit firewall permission"...

File: website_and_docs/content/documentation/grid/getting_started.zh-cn.md [hunk 4]
```diff
@@ -214,113 +203,83 @@ driver.quit();
...
 
-Failure to protect your Grid could result in one or more of the following occurring:
+未能保护您的网格可能会导致发生以下一种或多种情况：
```
Comment (alaahong) [inline website_and_docs/content/documentation/grid/getting_started.zh-cn.md]: How about to keep "Grid" just like what you did in almost above parts?

**Review by alaahong — CHANGES_REQUESTED** [review]
Thanks for your efforts. Almost good, can you take a look into the highlight parts?

File: website_and_docs/content/documentation/grid/getting_started.zh-cn.md (hunk 4 above)
Comment (iamxiaojianzheng) [reply website_and_docs/content/documentation/grid/getting_started.zh-cn.md]: In response to the points you mentioned, I have made some changes and psuh to the patch-1 branch.

File: website_and_docs/content/documentation/grid/getting_started.zh-cn.md (hunk 1 above)
Comment (iamxiaojianzheng) [reply website_and_docs/content/documentation/grid/getting_started.zh-cn.md]: fixed

File: website_and_docs/content/documentation/grid/getting_started.zh-cn.md (hunk 2 above)
Comment (iamxiaojianzheng) [reply website_and_docs/content/documentation/grid/getting_started.zh-cn.md]: fixed

File: website_and_docs/content/documentation/grid/getting_started.zh-cn.md (hunk 3 above)
Comment (iamxiaojianzheng) [reply website_and_docs/content/documentation/grid/getting_started.zh-cn.md]: fixed

File: website_and_docs/content/documentation/webdriver/troubleshooting/errors.zh-cn.md [hunk 5]
```diff
@@ -8,93 +8,66 @@ description: >
...
 ## 过时元素引用的异常 (Stale Element Reference Exception)
 
-元素先前的定位会过时，
-导致当前无法访问.
-元素不会自动重新定位;
-驱动程序为元素创建引用ID，
-并且有一个特定的位置，以期在 DOM 中找到.
-如果在当前 DOM 中找不到元素,
-任何使用该元素的操作都将导致此异常.
+当成功定位到元素时，WebDriver会为其设置一个引用ID作为标记，如果由于上下文环境发生变化，导致之前元素的位置发生了变化或者无法找到了，WebDriver并不会自动重新定位，任何使用之前元素所做的操作将报错该异常。
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/troubleshooting/errors.zh-cn.md]: As Selenium suggestion, you should not put all into one line.
reference --> https://www.selenium.dev/documentation/about/contributing/#step-4-commit

File: website_and_docs/content/documentation/webdriver/troubleshooting/errors.zh-cn.md [hunk 6]
```diff
@@ -8,93 +8,66 @@ description: >
...
+元素对象是针对特定的上下文存储的，因此如果您切换到不同的上下文，比如不同的 `Window` 或不同的 `frame` 或 `iframe` 元素引用仍然有效，但暂时无法访问。在这种情况下，重新定位元素无济于事，因为它在当前上下文中不存在。要解决此问题，您需要确保在使用该元素之前切换回正确的上下文。
 
 **页面已变更**
 
-这种情况是指您不仅更改了上下文, 还导航到了另一个页面, 
-并破坏了元素所在的上下文.
-您不能只是将其从当前上下文中重新定位,
-并且您无法切换回有效的活动上下文.
-如果这是原因, 则对于您的错误, 您必须导航回正确的位置并重新定位它.
+这种情况发生在您不仅更改了上下文，而且导航到另一个页面并破坏了元素所在的上下文。您无法仅从当前上下文重新定位它，也无法切换回元素有效的活动上下文。如果这是您的错误原因，您必须回到正确的位置并重新定位元素。
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/troubleshooting/errors.zh-cn.md]: Seems no mandatory change on this error file?

**Review by alaahong — COMMENTED** [review]
Glad to see such contributions, just need follow the Selenium rule and no need to refresh existing file if no content changed or missed.

File: website_and_docs/content/documentation/webdriver/troubleshooting/errors.zh-cn.md (hunk 5 above)
Comment (iamxiaojianzheng) [reply website_and_docs/content/documentation/webdriver/troubleshooting/errors.zh-cn.md]: OK

**Review by alaahong — APPROVED** [review]

**Comment by harsha509** [comment]
Thank you for the review @alaahong !

**Pull request merged**